docs/examples/19_particle_tracking_prt.ipynb

```diff
@@ -184,11 +233,312 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create new subdir for PRT backwards model\n",
+    "prt_ws_bw = Path(dsr.attrs[\"model_ws\"]) / \"prt_bw\"\n",
+    "\n",
+    "# Create a new Simulation object\n",
+    "simprt_bw = nlmod.sim.sim(dsr, sim_ws=prt_ws_bw)\n",
+    "_ = nlmod.sim.tdis(dsr, simprt_bw)\n",
+    "\n",
+    "# Add PRT model\n",
+    "prt_bw = nlmod.prt.prt(dsr, simprt_bw, modelname=dsr.model_name)\n",
+    "\n",
+    "# Add PRT model\n",
+    "_ = nlmod.prt.disv(dsr, prt_bw)\n",
+    "\n",
+    "# DIS: discretization package\n",
+    "_ = nlmod.prt.mip(dsr, prt_bw, porosity=0.3)\n",
+    "\n",
+    "# PRP: particle release package\n",
+    "_ = nlmod.prt.prp(\n",
+    "    dsr,\n",
+    "    prt_bw,\n",
+    "    packagedata=releasepts,\n",
+    "    perioddata={0: [\"FIRST\"]},\n",
+    ")\n",
+    "\n",
+    "# OC: output control\n",
+    "trackcsvfile_prt_bw = f\"{prt_bw.name}bw.trk.csv\"\n",
+    "_ = nlmod.prt.oc(dsr, prt_bw, trackcsv_filerecord=trackcsvfile_prt_bw)\n",
+    "\n",
+    "# FMI: flow model interface\n",
+    "fmi_bw_packagedata = [\n",
+    "    (\"GWFHEAD\", hds_bw_path),\n",
+    "    (\"GWFBUDGET\", cbc_bw_path),\n",
+    "]\n",
+    "_ = nlmod.prt.fmi(dsr, prt_bw, packagedata=fmi_bw_packagedata)\n",
+    "\n",
+    "# EMS: explicit model solution\n",
+    "_ = nlmod.sim.ems(simprt_bw, model=prt_bw)\n",
+    "\n",
+    "# Write and run the simulation\n",
+    "simprt_bw.write_simulation()\n",
+    "success_bw_prt, _ = simprt_bw.run_simulation()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Load pathline data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pathlines_bw = nlmod.prt.read_pathlines(\n",
+    "    prt_ws_bw / trackcsvfile_prt_bw, icell2d=dsr.icell2d.size\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Get pathlines of particles that started in a well (backward)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pathlines_bw0 = pathlines_bw.query(\"ireason==0\")\n",
+    "well_pids = []\n",
+    "for _, row in wel.stress_period_data.dataframe[0].iterrows():\n",
+    "    cellid_layer = row[\"cellid_layer\"]\n",
+    "    icell2d = row[\"cellid_cell\"]\n",
+    "    well_pids.append(\n",
+    "        pathlines_bw0.query(f\"ilay=={cellid_layer} and icell2d=={icell2d}\")\n",
+    "        .loc[:, \"pid\"]\n",
+    "        .to_numpy()[0]\n",
+    "    )\n",
+    "pathlines_bww = pathlines_bw[pathlines_bw[\"pid\"].isin(well_pids)]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Plot results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "f, ax = plt.subplots(figsize=(8, 8))\n",
```

Comment:
You could also just run the model with particles starting in the well if you only show that.

Reply:
Yes it feels a bit awkward. I wanted to do an advanced selection of starting points. But nothing interesting to show because all particles start from the river.

Reply:
Yes, I don't know if the PRT is able to this but for modpath there are some options to start at all cell faces for backward tracking. That might show a bit more than just these two lines. See also this notebook:

https://nlmod.readthedocs.io/en/stable/examples/10_modpath.html

Reply:
I'll try to make an interesting figure with all particles as well.

Reply:
I added a similar figure to the modpath